add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $title = $_POST['title'];
```

Comment:
Refactor these functionalities to use prepared statements to prevent SQL injection.
Consider implementing a `Book` class and a `BookRepository` for better structure and security by adhering to MVC patterns.
Separating PHP logic from HTML makes it easier to maintain.
Implement unit tests to validate functionalities.